petab/lint.py

```diff
@@ -39,11 +35,12 @@ def check_condition_df(df):
             f"Condition table has wrong index {df.index.name}."
             "expected 'conditionId'.")
 
-    assert_no_trailing_whitespace(df.index.values, "conditionId")
+    assert_no_leading_trailing_whitespace(df.index.values, "conditionId")
```

Comment:
would actually be helpful to have a method that can fix such simple style issues and save to file.

Reply:
Not sure of often this will be helpful. Mostly comes from inappropriate copy and pasting...

Reply:
yeah, ok. nothing needing to be implemented right now anyway